src/20_Components/amp-o2-player.html

```diff
@@ -0,0 +1,38 @@
+<!--
+  #### Introduction
+
+  Embed o2 player in your AMP HTML files.
+-->
+<!-- -->
+<!doctype html>
+<html ⚡>
+<head>
+  <meta charset="utf-8">
+  <!-- #### Setup -->
+  <!--
+  Import the `amp-o2-player` component in the header.
+  -->
+  <script async custom-element="amp-o2-player" src="https://cdn.ampproject.org/v0/amp-o2-player-0.1.js"></script>
+  <link rel="canonical" href="https://ampbyexample.com/components/amp-o2-player/" >
+  <meta name="viewport" content="width=device-width,minimum-scale=1,initial-scale=1">
+  <style amp-boilerplate>body{-webkit-animation:-amp-start 8s steps(1,end) 0s 1 normal both;-moz-animation:-amp-start 8s steps(1,end) 0s 1 normal both;-ms-animation:-amp-start 8s steps(1,end) 0s 1 normal both;animation:-amp-start 8s steps(1,end) 0s 1 normal both}@-webkit-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@-moz-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@-ms-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@-o-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}</style><noscript><style amp-boilerplate>body{-webkit-animation:none;-moz-animation:none;-ms-animation:none;animation:none}</style></noscript>
+  <script async src="https://cdn.ampproject.org/v0.js"></script>
+</head>
+<body>
+
+<!-- #### Basic Usage -->
+<!--
+To embed o2 player in your AMP HTML page you need to add amp-02-player tag with two mandatory parameters data-pid, and one of
+data-vid or data-bcid, other supported parameters are: data-bid, data-macro, layout (we recommend to use responsive),
+ width and height, last two define aspect ratio of the player.
```

Comment:
Can you highlight tag and attribute names, using "``", e.g. add `amp-02-player` tag